app/models/course_user.rb

```diff
@@ -32,6 +32,9 @@ def experience_points
   # TODO: Remove the map when Rails 5 is released.
   scope :staff, -> { where(role: STAFF_ROLES.map { |x| roles[x] }) }
 
+  scope :pending, -> { where(workflow_state: :pending) }
+  scope :approved, -> { where(workflow_state: :approved) }
```

Comment:
@lowjoel, do you want to remove this ? Workflow gem provided scopes already: https://github.com/geekq/workflow/tree/master#scopes